src/test/java/ICS_Donuts_Special.java

```diff
@@ -0,0 +1,141 @@
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+        import org.junit.Assert;
+        import org.openqa.selenium.*;
+        import org.openqa.selenium.chrome.ChromeDriver;
+        import org.openqa.selenium.chrome.ChromeOptions;
+
+        import java.util.HashMap;
+        import java.util.List;
+        import java.util.Map;
+
+
+public class ICS_Donuts_Special {
+    public static void main(String[] args) throws InterruptedException {
+        donutsPlpTest("desktop");
+        donutsPlpTest("tablet");
+        donutsPlpTest("mobile");
+    }
+
+    public static void donutsPlpTest(String size) throws InterruptedException {
+        WebDriverManager.chromedriver().setup();
+
+        WebDriver driver = driverSetup(size);
+
+        Dimension desktop = new Dimension(1366, 768);
+        Dimension tablet = new Dimension(768, 1024);
+        Dimension mobile = new Dimension(400, 800);
+        if (size.equals("desktop")){
+            driver.manage().window().setSize(desktop);
+        }else if (size.equals("tablet")){
+            driver.manage().window().setSize(tablet);
+        }else{
+            driver.manage().window().setSize(mobile);
+        }
+
+        driver.get("https://icecreamshop-frontend-rkbemzbp2q-ez.a.run.app/donutsForSpecialNeeds");
+
+        //fetching list web elements titles
+
+        List<WebElement> titlesList = driver.findElements(By.className("ItemSection_title___WbS6"));
+
+        //fetching top selling title element
+        WebElement topSellingTitle = titlesList.get(0);
+        String topSellingTitleText = topSellingTitle.getText();
+        Assert.assertEquals("TOP SELLING DONUTS FOR SPECIAL NEEDS", topSellingTitleText);
+
+
+        //fetching the top selling donuts cards into a list
+        WebElement topSellingSection = driver.findElement(By.className("ItemSection_section__3zVJh"));
+
+        Thread.sleep(1000);
```

Comment:
instead of thread sleep you can use something like this:
WebDriverWait wait = new WebDriverWait(driver, 20); // 20 being the time unit as how long you want the wait to take place
 wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector(locator_name)));
        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(locator_name)));